web_src/js/features/repo-home.ts

```diff
@@ -7,141 +7,144 @@ import {fomanticQuery} from '../modules/fomantic/base.ts';
 const {appSubUrl} = window.config;
 
 export function initRepoTopicBar() {
-  const mgrBtn = document.querySelector('#manage_topic');
-  if (!mgrBtn) return;
+  for (const viewType of ['desktop', 'mobile']) {
+    const mgrBtn = document.querySelector<HTMLButtonElement>(`#manage_topic_${viewType}`);
+    if (!mgrBtn) continue;
 
-  const editDiv = document.querySelector('#topic_edit');
-  const viewDiv = document.querySelector('#repo-topics');
-  const topicDropdown = editDiv.querySelector('.ui.dropdown');
-  let lastErrorToast;
+    const editDiv = mgrBtn.parentNode.querySelector(`#topic_edit_${viewType}`);
+    const viewDiv = mgrBtn.parentNode.querySelector(`#repo-topics-${viewType}`);
```

Comment:
It is too hacky and fragile. Use some more general approach like CSS selectors instead.

Reply:
Done in a4b0b0ee8d4166ebdbc620e7975bd2cbd7ae1aaa